models/src/main/java/com/foodie/FoodieBaseService.java

```diff
@@ -86,4 +88,30 @@ default Page<T> findAllByDeletedAtIsNull(int page, int size, String sort) {
         List<T> content = lazyStreamable.toList();
         return new PageImpl<>(content, pageable, content.size());
     }
-}
+
+    private void updateFields(T sourceEntity, T targetEntity) {
+        Class<?> clazz = sourceEntity.getClass();
+        Field[] fields = clazz.getDeclaredFields();
+        
+        for (Field field : fields) {
+            if (!isFieldIgnored(field)) {
+                field.setAccessible(true);
```

Comment:
This violates the encapsulation principle

Reply:
For now it is needed